cheroot/test/helper.py

```diff
@@ -40,7 +41,7 @@ class CherootWebCase(webtest.WebCase):
     scheme = 'http'
 
     available_servers = {
-        'wsgi': cheroot.server.WSGIServer,
+        'wsgi': cheroot.wsgi.WSGIServer,
```

Comment:
Right, I get your point.

So should I just make a new commit on this one, or pull another request?

I think the latter is better?